docs/finalproject/prep.md

```diff
@@ -48,9 +48,9 @@ All code should be done _in public on the trainees' Githubs_, unless there is a
 
 Before class starts volunteer and trainee teams should have been assigned.
 
-You can find a template for organising the teams [here](https://docs.google.com/spreadsheets/d/13WGRmeUZoCZfPV3joO4q69GLIaAiYFnrh9WmZ51FrE8/edit#gid=0)
+You can find a template for organising the teams[here](https://docs.google.com/spreadsheets/d/16vSSJgzCZJKF-2pwuBTkKjJJJ9i1CGRqMbYB-HEO5mo/edit?usp=sharing). 
```

Comment:
```suggestion
Please make a copy of this [template spreadsheet](https://docs.google.com/spreadsheets/d/16vSSJgzCZJKF-2pwuBTkKjJJJ9i1CGRqMbYB-HEO5mo/edit?usp=sharing) to organise the teams and track the progress of the projects.
```

Links must make sense out of context. This is important for accessibility. https://www.w3.org/QA/Tips/noClickHere